artman/config/proto/config_pb2.py

```diff
@@ -0,0 +1,569 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Have you thought about having artman generate this on-the-fly instead of checking it in?